src/main/java/io/github/thebusybiscuit/slimefun4/utils/RadiationUtils.java

```diff
@@ -0,0 +1,38 @@
+package io.github.thebusybiscuit.slimefun4.utils;
+
+import io.github.thebusybiscuit.slimefun4.api.player.StatusEffect;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import org.apache.commons.lang.Validate;
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.Player;
+
+import javax.annotation.Nonnull;
+import java.util.Objects;
+
+/**
+ * This class is a basic wrapper around the
+ * status effect.
+ *
+ * @author Semisol
+ *
+ * @see RadiationSymptom
+ */
+public class RadiationUtils {
+    private static final StatusEffect RADIATION_EFFECT = new StatusEffect(new NamespacedKey(Objects.requireNonNull(SlimefunPlugin.instance()), "radiation"));
+    public static void clearFromPlayer(@Nonnull Player p){
+        Validate.notNull(p, "The player cannot be null");
+        RADIATION_EFFECT.clear(p);
+    }
+    public static void addToPlayer(@Nonnull Player p, int exposure){
+        Validate.notNull(p, "The player cannot be null");
+        RADIATION_EFFECT.addPermanent(p, Math.max(RADIATION_EFFECT.getLevel(p).orElse(0) + exposure, 100));
+    }
+    public static void removeFromPlayer(@Nonnull Player p, int exposure){
+        Validate.notNull(p, "The player cannot be null");
+        RADIATION_EFFECT.addPermanent(p, Math.min(RADIATION_EFFECT.getLevel(p).orElse(0) - exposure, 0));
+    }
+    public static int getExposure(@Nonnull Player p){
+        Validate.notNull(p, "The player cannot be null");
+        return RADIATION_EFFECT.getLevel(p).orElse(0);
+    }
+}
```

Comment:
* Some formatting issues
* Missing constructor (util classes should never be publicly instantiable)
* Some redundancy (`RADIATION_EFFECT.getLevel(p).orElse(0)` is re-used VERY often, even though a helper method for it exists)
* No need for `Objects.requireNonNull()` here.
* Also renamed the methods a bit

```suggestion
public class RadiationUtils {

    private static final StatusEffect RADIATION_EFFECT = new StatusEffect(new NamespacedKey(SlimefunPlugin.instance(), "radiation"));

    private RadiationUtils() {}

    public static void clearExposure(@Nonnull Player p) {
        Validate.notNull(p, "The player cannot be null");
        RADIATION_EFFECT.clear(p);
    }

    public static int getExposure(@Nonnull Player p) {
        Validate.notNull(p, "The player cannot be null");
        return RADIATION_EFFECT.getLevel(p).orElse(0);
    }

    public static void addExposure(@Nonnull Player p, int exposure) {
        Validate.notNull(p, "The player cannot be null");

        int level = Math.max(getExposure(p) + exposure, 100);
        RADIATION_EFFECT.addPermanent(p, level);
    }

    public static void removeExposure(@Nonnull Player p, int exposure) {
        Validate.notNull(p, "The player cannot be null");

        int level = Math.min(getExposure(p) - exposure, 0);
        RADIATION_EFFECT.addPermanent(p, level);
    }
}
```

Reply:
I will do this locally as other parts depend on it

Reply:
Done! Also I saw the response late due to me not checking my notifs too often.